src/firmware.cpp

```diff
@@ -340,8 +304,27 @@ static void process_serial() {
   serialBuffer += (char)serialInput;
 }
 
+
+void setup() {
+  // Setup the pins.
+  pinMode(LED_BUILTIN, OUTPUT);
+  for (int pin = 2; pin <= 13; ++pin) {
+    pinMode(pin, INPUT);
+  }
+
+  Serial.begin(9600);
+  Serial.setTimeout(5);
+
+  SERVOS.begin();
+  SERVOS.setPWMFreq(50);
+
+  Serial.write("# Booted\n");
```

Comment:
Previously this was all lowercase; is there anything which relies on an exact string match? (`interface.py` definitely did; I'm not sure whether `robotd` does)

Reply:
I did check, it appeared to only be in `interface.py`. `robotd` does not check for this string *at all*, never mind an exact string match.

Reply:
Under the old protocol anything which starts with a '#' is a comment. It is not permissible to assign programmatic semantics to comments.